metal/datasource_metal_metro.go

```diff
@@ -0,0 +1,67 @@
+package metal
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/packethost/packngo"
+)
+
+func dataSourceMetalMetro() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceMetalMetroRead,
+		Schema: map[string]*schema.Schema{
+			"id": {
```

Comment:
`id`, `country` and `name` are not input arguments - having "Optional" set is useless here

There are similar issues with fields in the facility datasource, probably my fault. Can you fix those too please?